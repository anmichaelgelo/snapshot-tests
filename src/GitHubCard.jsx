import {Button, Card} from 'react-bootstrap';

function GitHubCard() {
  return (
    <Card style={{ width: '18rem' }}>
      <Card.Img variant="top" src="/me.jpg" width={100} />
      <Card.Body>
        <Card.Title>anmichaelgelo</Card.Title>
        <Card.Text>Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...</Card.Text>
        <Button variant="primary">Go somewhere</Button>
      </Card.Body>
    </Card>
  );
}

export default GitHubCard;