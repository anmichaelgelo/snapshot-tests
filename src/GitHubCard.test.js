import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer'
import GitHubCard from './GitHubCard';

it('renders a snapshot of GitHubCard', () => {
  const tree = renderer
    .create(<GitHubCard />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

