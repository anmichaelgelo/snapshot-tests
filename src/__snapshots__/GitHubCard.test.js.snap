// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of GitHubCard 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="/me.jpg"
    width={100}
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      anmichaelgelo
    </div>
    <p
      className="card-text"
    >
      Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...
    </p>
    <button
      className="btn btn-primary"
      disabled={false}
      type="button"
    >
      Go somewhere
    </button>
  </div>
</div>
`;
